src/app.ts

```diff
@@ -41,7 +42,8 @@ export default async function (instance: FastifyInstance): Promise<void> {
     .register(fp(metaPlugin))
     .register(fp(passportPlugin))
     // need to be defined before member and item for auth check
-    .register(fp(authPlugin));
+    .register(fp(authPlugin))
+    .register(tagPlugin);
```

Comment:
Is there a reason why you register it here instead of next to the `memberServiceApi` ?